examples/disaggregated/slurm/benchmark/start_worker.sh

```
@@ -63,7 +72,7 @@
     elif [ "${role}" = "CTX" ]; then
         echo "nsys is not enabled on ctx_gpus"
     fi
-    trtllm-llmapi-launch ${nsys_prefix} \
+    trtllm-llmapi-launch ${numa_bind_cmd} ${nsys_prefix} \
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)